src/build.d

```diff
@@ -525,10 +525,10 @@ alias buildFrontendHeaders = makeRule!((builder, rule) {
     builder
         .name("cxx-headers")
         .description("Build the C++ frontend headers ")
-        .msg("(DC) CXX-HEADERS")
```

Comment:
see #13771

Reply:
This step is actually DMD specific since it uses the freshly built compiler to generate the headers, so I think it makes sense to prefix it with `(DMD)`.